<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetricsDialog</class>
 <widget class="QDialog" name="MetricsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>365</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableWidget" name="table"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Related to Syndrome</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="syndromesComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select a syndrome that maps to phenotypes.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Related to Phenotype</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="hpoComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Select a phenotype to see the associated metrics.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolButton" name="flipButton">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
	border: none;
}
QToolButton:hover{
	border: 1px solid #8f8f91;
	border-radius: 4px;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../FaceTools_res.qrc">
         <normaloff>:/icons/BELOW</normaloff>:/icons/BELOW</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="toolTip">
          <string>Restrict the list of phenotypes to just those detected on the current subject.</string>
         </property>
         <property name="text">
          <string>Matched Phenotypes Only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="matchedCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Restrict the list of phenotypes to just those detected on the current subject.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolButton" name="showMetricsButton">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Visualise Metrics</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
	border: none;
}
QToolButton:hover{
	border: 1px solid #8f8f91;
	border-radius: 4px;
}
QToolButton:checked{
	border: 1px solid #8f8f91;
	border-radius: 4px;
	background-color: rgb(211, 215, 207);
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../FaceTools_res.qrc">
         <normaloff>:/icons/METRICS</normaloff>:/icons/METRICS</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showChartButton">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Show Growth Curves</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
	border: none;
}
QToolButton:hover{
	border: 1px solid #8f8f91;
	border-radius: 4px;
}
</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../FaceTools_res.qrc">
         <normaloff>:/icons/CHART</normaloff>:/icons/CHART</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showPhenotypesButton">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Show Human Phenotype Ontology Terms.</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
	border: none;
}
QToolButton:hover{
	border: 1px solid #8f8f91;
	border-radius: 4px;
}
</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../FaceTools_res.qrc">
         <normaloff>:/icons/FACEQ</normaloff>:/icons/FACEQ</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showScanInfoButton">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Show Model Info</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
	border: none;
}
QToolButton:hover{
	border: 1px solid #8f8f91;
	border-radius: 4px;
}
</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../FaceTools_res.qrc">
         <normaloff>:/icons/MODEL_INFO</normaloff>:/icons/MODEL_INFO</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ignoreEthnicityCheckBox">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Ignore the ethnicity of the current subject when matching phenotypes or plotting on growth curves.</string>
       </property>
       <property name="text">
        <string>Ignore Ethnicity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../FaceTools_res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>MetricsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
